plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -26,11 +26,11 @@
 import io.trino.parquet.writer.ParquetSchemaConverter;
```

Comment:
Follow the guideline for commit comments for https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#commits-and-pull-requests

Reply:
thanks for information